### ISSUE TITLE
Parsing struct-typedefs with multiple names only recognizes the first name

### ISSUE THREAD

**Issue by seventh-chord** [slynch8/10x#2581]
I mostly use 10x to read code I'm unfamiliar with.
That code usually has all kinds of wierd declaration styles, and I've run into some cases which the 10x parser doesn't seem to handle.

---

E.g.:
```
typedef struct {
} foo, *pfoo;
typedef struct {
} *pbar, bar;
```

Here, only `foo` and `pbar` will be recognized as a type, and `pfoo` and `bar` will be ignored.

---

```
typedef struct {
} __attribute__((packed)) foo;

#define PACKED __attribute__((packed))
typedef struct {
} PACKED foo;
```

In these examples, `foo` also doesn't get recognized.

---

About dialog shows version 1.0.139, perpetual version 1.0.146.

**Comment by seventh-chord** [comment]
Thanks. But as far as I can tell, only the first case is fixed.

These three still don't parse:

```
typedef struct {
} *pbar, bar;

typedef struct {
} __attribute__((packed)) foo;

#define PACKED __attribute__((packed))
typedef struct {
} PACKED foo;
```

Checked with version 1.0.150.

**Comment by slynch8** [comment]
fixed (1.0.189)
